geonode/settings.py

```diff
@@ -633,6 +658,9 @@
                 'geonode.context_processors.resource_urls',
                 'geonode.geoserver.context_processors.geoserver_urls',
                 'geonode.themes.context_processors.custom_theme',
+
+                # IGRAC context processor
+                'igrac.context_processors.resource_urls',
```

Comment:
hi @zamuzakki this module does not exist

Reply:
Added context_processor.py